assets/js/components/Dashboard/AlertBanner.tsx

```diff
@@ -21,7 +21,8 @@ const AlertBanner: ComponentType<AlertBannerProps> = ({
   queueExpiration,
 }: AlertBannerProps) => {
   const { bannerAlert } = useScreenplayContext();
-  if (!bannerAlert) return null;
+  if (useLocation().pathname.includes("configure-screens") || !bannerAlert)
+    return null;
```

Comment:
Since this early return results in all of the code past line 25 being conditionally evaluated, the `useParams` on line 30 violates the "don't call hooks conditionally" rule<sup>1</sup>. Could you either:
1. move `const params = useParams();` up before this `if` statement, or
2. move this conditional up to the parent component, so that this component is simply not mounted if it shouldn't render anything. E.g.
   ```ts
   // in Dashboard component
   const { bannerAlert } = useScreenplayContext();
   const pathname = useLocation().pathname;
   const showBanner = bannerAlert && !pathname.includes("configure-screens");

   // in JSX returned by component
   {showBanner && <AlertBanner ... />}
   ```

(2) is preferable, since (1) leaves the potential for future bugs if a new hook call is added to this component later on.

<sup>1</sup> [Can you early return with React hooks?](https://stackoverflow.com/questions/54938236/can-you-early-return-with-react-hooks)

Reply:
Good catch. Totally missed that. The `Sidebar` wasn't violating the rule, but I moved that conditional to `Dashboard` so the conditional rendering is done at the same place.